core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/dot-experiments-list.module.ts

```diff
@@ -7,6 +7,7 @@ import { ConfirmDialogModule } from 'primeng/confirmdialog';
 import { ConfirmPopupModule } from 'primeng/confirmpopup';
 import { MenuModule } from 'primeng/menu';
 import { MultiSelectModule } from 'primeng/multiselect';
+import { RippleModule } from 'primeng/ripple';
```

Comment:
We're not adding this to the new design system but I'll check with Melissa.

Reply:
Perfect. I will reach her to confirm and remove it from Experiments

Reply:
Removed from all DotExperiments